changelog.d/9229.misc

```diff
@@ -0,0 +1 @@
+Fix a bug where `None` was passed to Synapse modules instead of an empty dictionary if an empty module `config` block was provided in the homeserver config.
```

Comment:
If this is a bug fix it should probably be a `.bugfix`? I could go either way on it though

Reply:
Yeah, it should probably be a bugfix, especially if the changelog starts that way :blush: